src/client/java/hadoop-daos/src/test/java/io/daos/fs/hadoop/DaosConfigFileTest.java

```diff
@@ -1,19 +1,5 @@
 /**
```

Comment:
The copyright is missing

Reply:
OK, the original file doesn't have Copyright, this ticket does not intend to add that.